### ISSUE TITLE
Is there a way to using Producer.commitableSink after a flow?

### ISSUE THREAD

**Issue by hanrw** [gzoller/LateKafka#5]
Hi 
 just see your code from gitter, what's i want to do is that deal message from kafka after that repush to kafka. i want ask suggestion or how to achieve as the code below.
Thanks.

```
  val src = Consumer.atMostOnceSource(consumerSettings.withClientId("client" + id))
  val commit = Flow[In].mapAsync(1) { msg =>
    Future {
      // msg.committableOffset.commit()
      syncInc()
      if (count == num) {
        println(s"[$id] Time ($count): " + (count / ((System.currentTimeMillis() - now) / 1000)) + " TPS")
      }
    }
  }
  val work = Flow[In].map { i => i } // a dummy step where real "work" would happen

  // src ~> work ~> commit ~> Sink.ignore
  src ~> work ~> commit ~> Producer.commitableSink(producerSettings) 
  ClosedShape
})
```

**Comment by hanrw** [comment]
Changed Consumer.atMostOnceSource  to Consumer.committableSource

**Comment by gzoller** [comment]
Yes, that should work.  Actually that code is in the test directory of my project.  LateKafka is a wrapper for a KafkaConsumer to provide a Source for an Akka stream.  (I found the official reactive-kafka library to be very slow for some reason.)  How you decide use that Source is up to you, and not prescribed by the library.  I've provided this sample in test for my use case, and am glad you found another way to use it.  Good luck!
